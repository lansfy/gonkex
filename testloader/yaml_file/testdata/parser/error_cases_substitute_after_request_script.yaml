- name: loader MUST fail if cannot substitute afterRequestScriptArgs to request body
  method: GET
  path: /some/path
  response:
    200: "aaabbb"
  afterRequestScript:
    path: '/some/path/{{ .bar }'
  cases:
    - afterRequestScriptArgs:
        bar: "otherscript1.sh"
