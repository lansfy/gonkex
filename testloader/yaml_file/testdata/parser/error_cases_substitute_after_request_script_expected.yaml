- Error: "process 'testdata/parser/error_cases_substitute_after_request_script.yaml': test 'loader MUST fail if cannot substitute afterRequestScriptArgs to request body #1': template: $.afterRequestScript.path:1: unexpected \"}\" in operand"
